embedded-kafka/src/main/scala/net/manub/embeddedkafka/EmbeddedKafkaConfig.scala

```diff
@@ -1,11 +1,30 @@
 package net.manub.embeddedkafka
 
-case class EmbeddedKafkaConfig(kafkaPort: Int = 6001,
-                               zooKeeperPort: Int = 6000,
-                               customBrokerProperties: Map[String, String] = Map.empty,
-                               customProducerProperties: Map[String, String] = Map.empty,
-                               customConsumerProperties: Map[String, String] = Map.empty)
+trait EmbeddedKafkaConfig {
+  val kafkaPort: Int
```

Comment:
those should all be `def`s. https://stackoverflow.com/questions/19642053/when-to-use-val-or-def-in-scala-traits